rules/windows/process_creation/proc_creation_win_net_use_mount_admin_share.yml

```
@@ -20,7 +23,7 @@ detection:
             - '\net1.exe'
         CommandLine|contains|all:
             - ' use '
-            - '\\\*\\*$' # (Specs) If some wildcard after a backslash should be searched, the backslash has to be escaped: \\*
+            - ' \\*\*$'
```

```suggestion
            - ' \\\\*\\*$'
```
- I assume you wanted to at the leading space
- I assume we want to match e.g. `\\<ip_or_hostname>\<drive_letter>$`  --> we could use a regex (but we need to escape the backslashes before the wildcard usage)

---

A wildcard will suffice in this case I would say no need for regex. Also, my notation is correct and easier to read as there is no escaping done for the `*` I wouldn't escape anything.

---

Assuming my second assumption is true: If you write `\\` it will be one literal `\`. If you want to have two literal backslashes in the beginning you have to use the escaped version. Similar the second wildcard. It is escaped in your case.

Before discussing: Is my 2nd assumption correct or are you trying to match some other pattern?

---

You're right, I assumed wrong. It's because the `\*` will be handled as an escape which creates a chain of escapes to handle it properly. Good catch.